page-objects/pages/administrationPages/ADMNSTRTNAdministerStudentsPage.js

```diff
@@ -4,65 +4,97 @@ const startPage = require('../generalPagesBeforeLogin/StartPageBeforeLogin');
 const loginPage = require('../generalPagesBeforeLogin/LoginPage');
 
 const waitHelpers = require('../../../runtime/helpers/waitHelpers');
+const elementHelpers = require('../../../runtime/helpers/elementHelpers');
 let oldPassword;
 
-const selectorAddStudentBtn = "button[data-testid='btn_add_student']";
-const selectorSetFirstName = "input[data-testid='create_student_input_firstname']";
-const selectorSetLastName = "input[data-testid='create_student_input_lastname']";
-const selectorSetEmail = "input[data-testid='create_student_input_email']";
-const selectorSendALinkBox = "input[data-testid='create_student_input_send_link']";
-const selectorNamesContainer = "tbody[data-testid='students_names_container']";
-const selectorConsentSubmitBtn = "button[data-testid='submit_consent']";
-const submitStudentCreateBtn = "div.modal.fade.add-modal.in button.btn-submit";
-const passwordFieldSel = "#passwd";
+const addStudentBtn = "button[data-testid='btn_add_student']";
+const firstNameInput = "input[data-testid='create_student_input_firstname']";
+const lastNameInput = "input[data-testid='create_student_input_lastname']";
+const emailInput = "input[data-testid='create_student_input_email']";
+const sendRegistrationLinkCheckbox = "input[data-testid='create_student_input_send_link']";
+const tableOfStudents = "tbody[data-testid='students_names_container']";
+const consentSubmitBtn = "button[data-testid='submit_consent']";
+const addStudentSubmitBtn = 'div.modal.fade.add-modal.in button.btn-submit';
+const passwordInput = '#passwd';
 
-module.exports = {
-    createNewPupil: async function (firstname, lastname, email) {
-        await waitHelpers.waitAndClick(selectorAddStudentBtn);
-        await waitHelpers.waitAndSetValue(selectorSetFirstName, firstname);
-        await waitHelpers.waitAndSetValue(selectorSetLastName, lastname);
-        await waitHelpers.waitAndSetValue(selectorSetEmail, email);
-        await this.setStudentsBirthdayScript();
-        await waitHelpers.waitAndClick(selectorSendALinkBox);
-        await waitHelpers.waitAndClick(submitStudentCreateBtn);
-    },
-    setStudentsBirthdayScript: async function () {
-        await driver.pause(1500);
-        await driver.execute('document.querySelector("#create_birthday").value = "13.08.1990"')
-    },
+//
+async function clickAddStudentBtn() {
+	await elementHelpers.clickAndWait(addStudentBtn);
+}
+
+async function setStudentFirstName(firstname) {
+	await waitHelpers.waitAndSetValue(firstNameInput, firstname);
+}
+
+async function setStudentLastName(lastname) {
+	await waitHelpers.waitAndSetValue(lastNameInput, lastname);
+}
+
+async function setStudentEmail(email) {
+	await waitHelpers.waitAndSetValue(emailInput, email);
+}
+
+async function clickOnSendRegistrationLinkCheckbox() {
+	await elementHelpers.click(sendRegistrationLinkCheckbox);
+}
 
```

Comment:
in which cases should we use .click and in which clickAndWait?

Reply:
Now we have 2 different methods. clickAndWait() we should use when the page is reloaded or a new page is opened after clicking on the element. click() method we should use when we want to just click on an element without waitUntilPageLoads

Reply:
Now we have 2 different methods. clickAndWait() we should use when the page is reloaded or a new page is opened after clicking on the element. click() method we should use when we want to just click on an element without waitUntilPageLoads. This PR is about changes in Administration Pages